runbotics/runbotics-orchestrator-ui/src/src-app/components/Tile/ProcessTile/ProcessTile.tsx

```
@@ -29,16 +38,77 @@ const ProcessTile: FC<ProcessTileProps> = ({ process }) => {
 
     const refProcessTileContent = useRef<HTMLDivElement>();
 
+    const { enqueueSnackbar } = useSnackbar();
+    const { translate } = useTranslations();
+    const dispatch = useDispatch();
+    const { allActiveMap: processInstanceMap } = useSelector(processInstanceSelector);
+    const processInstance = processInstanceMap[process.id]?.processInstance ?? null;
+    const [isProcessActive, setIsProcessActive] = useState(checkActiveProcess(processInstance));
+
     const handleRedirect = () => {
         if (hasBuildTabAccess) router.push(buildProcessUrl(process, ProcessTab.BUILD));
         else router.push(buildProcessUrl(process, ProcessTab.RUN));
     };
 
+    const handleProcessRun = (e: React.MouseEvent<HTMLButtonElement>) => {
+        e.preventDefault();
+        e.stopPropagation();
+
+        dispatch(processActions.startProcess({
+            processId: process.id
+        }))
+            .then(unwrapResult)
+            .then((response) => {
+                dispatch(
+                    processInstanceActions.updateOrchestratorProcessInstanceIdMap({
+                        processId: process.id,
+                        orchestratorProcessInstanceId: response.orchestratorProcessInstanceId,
+                    })
+                );
+
+                const lastRun = (new Date()).toISOString();
+                dispatch(processActions.updateProcessPage({ ...process, lastRun }));
+                setIsProcessActive(true);
+            })
+            .catch((error) => {
+                const translationKeyPrefix = 'Component.Tile.Process.Instance.Error';
+
+                const message = error?.message ?? translate(translationKeyPrefix);
```

redundant questionmark -> error.message ?? translate(translationKeyPrefix);

---

+ translationKeyPrefix is both prefix and default error message - a little misleading. Maybe translate(translationKeyPrefix + 'Default') ?

---

in that case translationKeyPrefix could contain cosing '.' already